src/store/network.js

```diff
@@ -106,14 +106,13 @@ export default {
                 try {
                     isUp = await NetworkService.isNetworkUp(selectedNetwork);
                 } catch {
-                    isUp = false;
+                    isUp = state.network.nodeFailedAttempts < 5;
```

Comment:
IMO better to store `5` in config

Reply:
Yep, thanks putting that in a global variable. Will maybe move that into an env file in the future